docs/website/download/index.md

```diff
@@ -6,47 +6,86 @@ children:
 - ../meta/versions.md
 ---
 
-## Latest Brooklyn Source Code Release
+<div class="row">
+<div class="col-md-6" markdown="1">
 
-[Download Apache Brooklyn {{ site.brooklyn-stable-version }} source release from our mirror sites](https://www.apache.org/dyn/closer.cgi/incubator/brooklyn/{{ site.brooklyn-stable-version }}/apache-brooklyn-{{ site.brooklyn-stable-version }}.tar.gz)
+<div class="panel panel-default">
+  <div class="panel-heading" markdown="1">
+#### **Recommended download:** Distribution
+  </div>
+  <div class="panel-body" markdown="1">
+A pre-built package that contains Apache Brooklyn and all of its dependencies in a single, easy-to-run package. Suitable for
+Linux and Windows servers and workstations that have Java 1.7 or newer.
```

Comment:
Do we definitely properly support Java 1.8? I don't think that has been tested as thoroughly. Perhaps we should say "recommended with Java 1.7"? No strong feelings.